### PR TITLE
Allow anybody to do cherrypicks option

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

File: experiment/cherrypicker/server.go [hunk 1]
```diff
@@ -165,15 +177,17 @@ func (s *Server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
...
-		ok, err := s.ghc.IsMember(org, commentAuthor)
-		if err != nil {
-			return err
-		}
-		if !ok {
-			resp := fmt.Sprintf("only [%s](https://github.com/orgs/%s/people) org members may request cherry picks. You can still do the cherry-pick manually.", org, org)
-			s.log.WithFields(l.Data).Info(resp)
-			return s.ghc.CreateComment(org, repo, num, plugins.FormatICResponse(ic.Comment, resp))
+		if !s.allowAll {
+			// Only members should be able to do cherry-picks.
```
Comment (mfojtik) [inline experiment/cherrypicker/server.go]: /nit mode on

so this isn't anybody, just members right? ;-)

Comment (mfojtik) [reply experiment/cherrypicker/server.go]: nvmd. read this wrong.

Comment (kargakis) [reply experiment/cherrypicker/server.go]: ;)

**Pull request merged**